lcls-twincat-pmps/Library/Library.tmc

```diff
@@ -69,31 +66,46 @@
me><Type Namespace="Tc2_System">T_MaxString</Type><BitSize>2048</BitSize></Parameter><Local><Name>IteratorCounter</Name><Type>UINT</Type><BitSize>16</BitSize></Local><Local><Name>ErrorMessage</Name><Type Namespace="Tc2_System">T_MaxString</Type><BitSize>2048</BitSize></Local><Local><Name>TestInstancePath</Name><Type Namespace="Tc2_System">T_MaxString</Type><BitSize>2048</BitSize></Local><Local><Name>FunctionCallResult</Name><Type>DINT</Type><BitSize>32</BitSize></Local><Local><Name>CycleCount</Name><Type>UDINT</Type><BitSize>32</BitSize></Local><Local><Name>TestWithThisNameAlreadyExists</Name><Type>BOOL</Type><BitSize>8</BitSize></Local><Local><Name>LowerCasedTestName</Name><Type Namespace="Tc2_System">T_MaxString</Type><BitSize>2048</BitSize></Local><Local><Name>TrimmedTestName</Name><Type Namespace="Tc2_System">T_MaxString</Type><BitSize>2048</BitSize></Local></Method><Method><Name>AssertArrayEquals_DWORD</Name><Parameter><Name>Expecteds</Name><Comment><![CDATA[ DWORD array with expected values]]></Comment><Type PointerTo="1" RpcArrayDim="1">DWORD</Type><BitSize>32</BitSize><Properties><Property><Name>variable_length_array</Name></Property><Property><Name>Dimensions</Name><Value>1</Value></Property></Properties></Parameter><Parameter><Name>Actuals</Name><Comment><![CDATA[ DWORD array with actual values]]></Comment><Type PointerTo="1" RpcArrayDim="1">DWORD</Type><BitSize>32</BitSize><Properties><Property><Name>variable_length_array</Name></Property><Property><Name>Dimensions</Name><Value>1</Value></Property></Properties></Parameter><Parameter><Name>Message</Name><Comment><![CDATA[ The identifying message for the assertion error]]></Comment><Type Namespace="Tc2_System">T_MaxString</Type><BitSize>2048</BitSize></Parameter><Local><Name>Equals</Name><Type>BOOL</Type><BitSize>8</BitSize></Local><Local><Name>SizeEquals</Name><Type>BOOL</Type><BitSize>8</BitSize></Local><Local><Name>Index</Name><Type>DINT</Type><BitSize>32</BitSize></Local><Local><Name>ExpectedString</Name><Type>STRING(80)</Type><BitSize>648</BitSize></Local><Local><Name>ActualString</Name><Type>STRING(80)</Type><BitSize>648</BitSize></Local><Local><Name>AlreadyReported</Name><Type>BOOL</Type><BitSize>8</BitSize></Local><Local><Name>TestInstancePath</Name><Type Namespace="Tc2_System">T_MaxString</Type><BitSize>2048</BitSize></Local><Local><Name>SizeOfExpecteds</Name><Type>DINT</Type><BitSize>32</BitSize></Local><Local><Name>SizeOfActuals</Name><Type>DINT</Type><BitSize>32</BitSize></Local><Local><Name>ExpectedDWordString</Name><Type>STRING(80)</Type><BitSize>648</BitSize></Local><Local><Name>ActualDWordString</Name><Type>STRING(80)</Type><BitSize>648</BitSize></Local><Local><Name>ExpectedsIndex</Name><Type>DINT</Type><BitSize>32</BitSize></Local><Local><Name>ActualsIndex</Name><Type>DINT</Type><BitSize>32</BitSize></Local></Method><Method><Name>AssertEquals_BYTE</Name><Parameter><Name>Expected</Name><Comment><![CDATA[ BYTE expected value]]></Comment><Type>BYTE</Type><BitSize>8</BitSize
```

Comment:
Comments like this would also be great in the docs somewhere (admittedly unrelated to the PR...)